src/index.html

```diff
@@ -5,8 +5,388 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#Home">
+          <img
+            src="./images/logo.png"
+            alt="Moto logo"
+          >
+        </a>
+      </div>
+
+      <nav class="header__nav">
```

Comment:
Navigation should be a separate block not just element of header, cause navigation can be not only in header.